client/src/utils/dateUtil.tsx

```diff
@@ -44,3 +50,37 @@ export const getHoursToDays = cacheWrapperForUnaryFunction((hours: number) => {
 
   return `${hours} hours`;
 });
+
+export const getTimeInSeconds = (endDate: string, startDate: string) => {
+  const differenceInMilliseconds = new Date(endDate).getTime() - new Date(startDate).getTime();
```

Comment:
Can we add safe checks for `startDate` and `endDate` here so that consumers need to to worry about conditionals